tests/test_bot.py

```diff
@@ -2076,6 +2076,7 @@ async def make_assertion(url, request_data: RequestData, *args, **kwargs):
         monkeypatch.setattr(bot.request, "post", make_assertion)
         assert await bot.do_api_request("camel_case")
 
+    @pytest.mark.filterwarnings("ignore::telegram.warnings.PTBUserWarning")
```

Comment:
why is this needed now when it wasn't needed before? Some for the other places

Reply:
I changed the warning of `Bot.do_api_request` from a DeprecationWarning to UserWarning since DeprecationWarning there didn't make sense, nothing was being deprecated there

Reply:
Ah, now I get it 👍 I used deprecationwarning there b/c it made sense to me to say "Using `do_api_request` for endpoint XY is now deprecated, since we have a dedicated method for that". But I'm not married to that